CharlieBackend.Api/Controllers/StudentsController.cs

```diff
@@ -143,19 +140,35 @@ public async Task<ActionResult> PutStudent(long studentId, [FromBody]UpdateStude
         }
 
         /// <summary>
-        /// Disabling of student
+        /// Disable student
```

Comment:
```suggestion
        /// Disable student's account
```
Please update for other roles too